Sources/mpc-core-kit-swift/tss.swift

```diff
@@ -324,7 +324,7 @@ extension MpcCoreKit {
         // generate a random nonce for sessionID
         let randomKey = try BigUInt(  Data(hexString:  curveSecp256k1.SecretKey().serialize() )! )
         let random = BigInt(sign: .plus, magnitude: randomKey) + BigInt(Date().timeIntervalSince1970)
-        let sessionNonce = TSSHelpers.base64ToBase64url( base64: TSSHelpers.hashMessage(message: random.serialize().toHexString()))
+        let sessionNonce = TSSHelpers.base64ToBase64url( base64: try TSSHelpers.hashMessage(message: random.serialize().toHexString()))
```

Comment:
Not going to make a difference here. 

Though when serializing BigInt, serialize() by itself would serialize the sign as well, so if you're only looking for positives then use magnitude instead. Also, it may drop leading zeroes.
 
```suggestion
        let sessionNonce = TSSHelpers.base64ToBase64url( base64: try TSSHelpers.hashMessage(message: random.magnitude.serialize().addLeading0sForLength64().hexString))
```

Reply:
done